source/_integrations/nest.markdown

```diff
@@ -299,6 +299,28 @@ All Google Nest Cam models and the Google Nest Hello Video Doorbell support [Dev
 
 See [Automating Home Assistant](/getting-started/automation/) for the getting started guide on automations or the [Automation](/docs/automation/) documentation for full details.
 
+### Example
+
+This automation will trigger when a `nest_event` event type with a type of `camera_motion` is received from the specified `device_id`.
+
+```yaml
+alias: motion alert
+trigger:
+  - platform: event
```

Comment:
I looked closer and realized: I think we should encourage use of device triggers instead of event triggers.  The examples for `nest` are very similar, but i think my impression is device triggers are the preferred approach and they work better in the UI anyway.
 
```
trigger:
  - platform: device
    device_id: YOUR_DEVICE_ID
    domain: nest
    type: doorbell_chime
```

Reply:
Yes, however, device triggers are generally for use with the UI, not in bare YAML (but could be done, but we don't have it anywhere).

Reply:
OK, so i guess the yaml for the action is the part that is still relevant.  But then maybe just blueprint support for device triggers is the answer...